utils/hct/hctdb.py

```diff
@@ -6680,6 +6680,12 @@ def add_pass(name, type_name, doc, opts):
                     "t": "unsigned",
                     "d": "Unrolled size limit for loops with an unroll(full) or unroll_count pragma.",
                 },
+                {
+                    "n": "StructurizeLoopExits",
```

Comment:
Can you help me understand what this change does?  Does the test above provide coverage for this change?

Reply:
This script generates some boilerplate code for the pass arguments. I'll have to double check what exact code it generates, but I think it's automatically called by cmake.

Reply:
IIUC, without this change you can’t set `StructureizeLoopExits` on the `dxopt` command line, which the test case above does.